webbpsf/webbpsf_core.py

```diff
@@ -2671,11 +2673,11 @@ def _addAdditionalOptics(self, optsys, oversample=2):
         shift_x, shift_y = self._get_pupil_shift()
         rotation = self.options.get('pupil_rotation', None)
 
-        # NIRCam as-built weak lenses, from WSS config file, PRDOPSFLT-027
-        WLP4_diversity = 8.3443  # microns
-        WLP8_diversity = 16.5932  # microns
-        WLM8_diversity = -16.5593  # microns
-        WL_wavelength = 2.12  # microns
+        # NIRCam as-built weak lenses, from WSS config file, PRDOPSFLT-027 # TODO - Okay to delete these unused defines?
+        # WLP4_diversity = 8.3443  # microns
+        # WLP8_diversity = 16.5932  # microns
+        # WLM8_diversity = -16.5593  # microns
+        # WL_wavelength = 2.12  # microns
```

Comment:
Yeah, I think these values were used in a prior version of the code but are not currently used, so OK to delete.